### PR TITLE
Improve Governance consensus rules

### PR DESCRIPTION
**Pull request by JSKitty**
## Abstract

This proposal fixes a handful of small areas where MPW deviated from PIVX Core's Governance consensus rules, those being:
- The 'Budget Fee' finalisation confirms: MPW now uses [PIVX Core's rules](https://github.com/PIVX-Project/PIVX/blob/7b7803c57b6b59995c54ff8ad40080c5b45009da/src/budget/budgetmanager.cpp#L1578) for Mainnet and Testnet.
- The 'Budget Fee' confirmation time (24h rule), for this, we use [IsEstablished](https://github.com/PIVX-Project/PIVX/blob/7b7803c57b6b59995c54ff8ad40080c5b45009da/src/budget/budgetproposal.cpp#L197) from the `getbudgetinfo` results.
- - *I would have preferred computing confirm time in-MPW to reduce RPC reliance, but it felt overkill for a single bool.*
- Fixed how MN votes are computed, now MPW uses the [exact same calculation](https://github.com/PIVX-Project/PIVX/blob/7b7803c57b6b59995c54ff8ad40080c5b45009da/src/budget/budgetproposal.cpp#L211C36-L211C36) without rounding, which stops Testnet `0 vote` proposals from "Passing" due to Testnet's low MN count.

When a proposal falls under the '24h rule' (on Mainnet, it's 5m on Testnet) then the Governance Dashboard will signal this with a "Too Young" message, and will not allow it to pass or be funded.
image

---

## Testing
To test this PR, it's suggested to attempt these user flows, or variations of these:
- Compare PIVX.org with MyPIVXWallet (this PR), ensure the Passing and Funding states 100% match.

If any errors are found, the PR works unexpectedly, or you have viable suggestions to improve the UX or functionality of the PR, let me know!

---

### CODE REVIEW

**Review by panleone — COMMENTED** [review]
Looks good, just one comment

File: scripts/global.js [hunk 1]
```diff
@@ -2224,12 +2229,25 @@ async function renderProposals(arrProposals, fContested) {
...
+                strFundingStatus = translation.proposalFunded;
+            } else {
+                strStatus = translation.proposalFailing;
+                strFundingStatus = translation.proposalNotFunded;
+            }
+        } else {
+            // Scenario 2: Proposal is not established (nor funded)
+            strStatus = translation.proposalFailing;
+            strFundingStatus = translation.proposalTooYoung;
+        }
```
[the remark this answers is not in the document]
Comment (JSKitty) [reply scripts/global.js:2250]: Looks good to me, adding... :hourglass_flowing_sand:

**Review by panleone — APPROVED** [review]
tACK 497263d86950def571a4ee4930b7a1b82c6d5b46

**Review by Duddino — APPROVED** [review]
tACK

**Review by Liquid369 — APPROVED** [review]
tACK 497263d86950def571a4ee4930b7a1b82c6d5b46

**Pull request merged**